utils/redis/client.js

```diff
@@ -0,0 +1,107 @@
+import Redis from 'ioredis';
+
+import {
+	getAuctionHouse,
+	getSearchConnectedRealm,
+	getToken,
+} from '../clients/blizzard/client';
+
+import cacheFormatAuctionHousesData from '../formatData/cache/auctionHouse';
+import cacheFormatRealmData from '../formatData/cache/realm';
+
+import getAllItemInfo from '../db/getAllItem';
+import getAllRelevantItemInfo from '../db/getAllRelevantItemInfo';
+
+export const redis = new Redis(process.env.REDIS_URL);
```

Comment:
I think the `export` here is unnecessary until you actually use it in other files.